pylint/test/functional/useless_object_inheritance.py

```diff
@@ -0,0 +1,21 @@
+"""Check if a class inherits from object.
+In python3 every class implicitly inherits from object, therefore give refactoring message to
+ remove object from bases"""
+# pylint: disable=no-init, invalid-name, missing-docstring, too-few-public-methods
+# pylint: disable=inconsistent-mro
+import abc
+
+class A(object):    # [useless-object-inheritance]
+    pass
+
+class B:
+    pass
+
+class C(B, object): # [useless-object-inheritance]
```

Comment:
Can you add a test where we inherit from a class, say `A` and make sure that the inherited class does not emit this error? e.g.
```
class A(object):
    pass
class B(A): # should not emit
    pass
```

Reply:
added this test case.